assignments/array-methods.js

```diff
@@ -55,29 +55,67 @@ const runners = [{"id":1,"first_name":"Charmain","last_name":"Seiler","email":"c
 
 // ==== Challenge 1: Use .forEach() ====
 // The event director needs both the first and last names of each runner for their running bibs.  Combine both the first and last names into a new array called fullName. 
+
 let fullName = [];
-console.log(fullName);
+runners.forEach(list => {
+	fullName.push(`${list.first_name} ${list.last_name}`)
```

Comment:
perfect way to do this. remember that you don't need curly braces if you are only returning one line.

```
runners.forEach(list =>
  fullName.push(`${list.first_name} ${list.last_name}`)
);
```